catboost/libs/model/coreml_helpers.cpp

```diff
@@ -198,45 +393,87 @@ void NCatboost::NCoreML::ConvertCoreMLToCatboostModel(const Model& coreMLModel,
     int approxDimension = ensemble.numpredictiondimensions();
 
     TVector<TVector<const TreeEnsembleParameters::TreeNode*>> treeNodes;
-    TVector<TVector<TFloatSplit>> trees;
+    TVector<TVector<TModelSplit>> treesSplits;
```

Comment:
Comment about different fields meaning (FloatFeature.FloatFeature, OneHotFeature.CatFeatureIdx, OneHotFeature.Value) when filling this structure in ProcessOneTree is necessary.